lib/rubocop/cop/cop.rb

```diff
@@ -132,6 +132,12 @@ def display_cop_names?
           config['AllCops'] && config['AllCops']['DisplayCopNames']
       end
 
+      def display_style_guide?
+        style_guide_url &&
+          (debug? || @options[:display_style_guide] ||
```

Comment:
It mirrors the behavior of debug with respect to `:display_cop_names`. I personally like the verbosity—and usually expect insane verbosity when using a debug option with most tools—but I can nix it if you think it's too much.